CONTRIBUTING.md

```diff
@@ -88,7 +93,12 @@ for version in 4.0 4.1 4.2 master; do
   else
     # red in ANSI
     echo -e "\033[31m **** Tests failed against Rails ${RAILS_VERSION} **** \033[0m"
-  fi
+    read -p '[Enter] any key to continue, [q] to quit...' prompt
+    if [ "$prompt" = 'q' ]; then
+      unset RAILS_VERSION
+      exit 1
+    fi
```

Comment:
@remear totally understand your thoughts on leaving this auto-pilot.  My only counter argument is, that this script is really nice to get a new contributor going - and jumping on errors right away (for me) was nice.  As an experienced contributor, you probably have some more advanced tools in your belt and/or can just comment out L96 to skip the prompt.